versioned_docs/version-0.1.0/site/site-model.md

```diff
@@ -0,0 +1,70 @@
+|------------------------------------------------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `scopeWhereUptimeCheckEnabled(Builder $query)`         | Builder        | A query scope that filters the query to include only websites with uptime checks enabled.                                                                                                                       |
+| `scopeWhereSslCertificateCheckEnabled(Builder $query)` | Builder        | A query scope that filters the query to include only websites with SSL certificate checks enabled.                                                                                                              |
+| `scopeWhereIsNotOnMaintenance(Builder $query)`         | Builder        | A query scope that filters the query to include only websites that are not currently on maintenance.                                                                                                            |
+| `url()`                                                | Attribute      | Returns an **Attribute** instance that represents the URL of the website.                                                                                                                                           |
+| `uptimeCheck()`                                        | HasOne         | Returns a **HasOne** relationship between the **Site** model and the **UptimeCheck** model. This method allows you to retrieve the uptime check associated with the website.                                                |
+| `sslCertificateCheck()`                                | HasOne         | Returns a **HasOne** relationship between the **Site** model and the **SslCertificateCheck** model. This method allows you to retrieve the SSL certificate check associated with the website.                               |
+| `exceptionLogs()`                                      | HasManyThrough | Returns a **HasManyThrough** relationship between the **Site** model and the **ExceptionLog** model through the **ExceptionLogGroup** model. This method allows you to retrieve the exception logs associated with the website. |
+| `exceptionLogGroups()`                                 | HasMany        | Returns a **HasMany** relationship between the **Site** model and the **ExceptionLogGroup** model. This method allows you to retrieve the exception log groups associated with the website.                                 |
+| `newCollection(array $models = [])`                    | SiteCollection | Overrides the **newCollection** method to return a custom **SiteCollection** instance. This method allows you to customize the collection returned when querying the **Site** model.                                        |
+
```

Comment:
Add new line.

Reply:
@rpereira-tae no entiendo este

Reply:
Add a new line, a break line, you can use `<br />`